kit/src/main/java/com/oracle/javafx/scenebuilder/kit/skeleton/SkeletonContext.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2016, 2021, Gluon and/or its affiliates.
+ * Copyright (c) 2012, 2014, Oracle and/or its affiliates.
```

Comment:
New files need to have only the year when they were created. Therefore the above 2 lines can be replaced with:

```
Copyright (c) 2021, Gluon and/or its affiliates.
```

Reply:
Fixed.